runelite-client/src/main/java/net/runelite/client/plugins/bosstimer/Boss.java

```diff
@@ -57,7 +57,8 @@
 	CERBERUS(NpcID.CERBERUS, 8400, ChronoUnit.MILLIS, ItemID.HELLPUPPY),
 	THERMONUCLEAR_SMOKE_DEVIL(NpcID.THERMONUCLEAR_SMOKE_DEVIL, 8400, ChronoUnit.MILLIS, ItemID.PET_SMOKE_DEVIL),
 	KRAKEN(NpcID.KRAKEN, 8400, ChronoUnit.MILLIS, ItemID.PET_KRAKEN),
-	KALPHITE_QUEEN(NpcID.KALPHITE_QUEEN_965, 30, ChronoUnit.SECONDS, ItemID.KALPHITE_PRINCESS);
+	KALPHITE_QUEEN(NpcID.KALPHITE_QUEEN_965, 30, ChronoUnit.SECONDS, ItemID.KALPHITE_PRINCESS),
+	DUSK(NpcID.DUSK_7889, 120, ChronoUnit.SECONDS, ItemID.NOON);
```

Comment:
`2, ChronoUnit.MINUTES`?

Reply:
Meant to change that as I was going to take a different approach, though, I didn't.